src/Tooltip.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+class Tooltip extends Component {
+  componentDidMount () {
+    $('.tooltipped').tooltip();
```

Comment:
e.g. I am using a custom component as child which is called `<Button>`.
```
import { Button, Tooltip } from 'react-materialize;
<Tooltip><Button waves='light'>test</Button></Tooltip>
````
It will place `data-tooltip-id` into the component but not inside the props and not functioning.
It only happens when you are not using html element as a child.
But in `$('.tooltipped').tooltip()` way, it will execute the code and show it properly.
```
context: {}
**data-tooltip-id: "1db47f5e-5064-450b-35d8-9086538c32ab"**
props: {
  children: "Test"
  data-delay: 30
  data-html: false
  data-position: "top"
  data-tooltip: "Test"
  ...
}
```